File: JobAggregator/JobAggregator.Core/Entities/Resume.cs
using JobAggregator.Core.Enum;

namespace JobAggregator.Core.Entities;

public class Resume : Entity
{
    public int UserId { get; set; }
    public string? Experience { get; set; }
    public string? Links { get; set; }
    public int Age { get; set; }
    public Gender Gender { get; set; }
    public string? Photo { get; set; }
    public List<Language>? Languages { get; set; }
    public string? Education { get; set; }
    public int CountViews { get; set; }
    public int Priority { get; set; }
    public DateTime Created { get; set; }
    public List<Skill>? Skills { get; set; }
}

File: JobAggregatorFront/src/app/models/resume.ts
import { Gender } from "./enums/gender";
import { Handbook } from "./handbook";
import { Language } from "./language";

export interface Resume{
    id: number;
    userId: number;
    experience?: string | null;
    links: string | null;
    age: number;
    gender: Gender;
    photo?: string | null;
    languages: Language[];
    education?: string | null;
    countViews: number;
    priority: number;
    created: Date;
    skills: Handbook[];
}